transport/handler/routes.go

```diff
@@ -67,6 +67,7 @@ func (h *Handler) setupRoutes() {
 					r.Use(middleware.HandleResourceID("release_id", util.ContextSetReleaseID))
 					r.Get("/", middleware.RequireAuthUser(h.getRelease))
 					r.Patch("/", middleware.RequireAuthUser(h.updateRelease))
+					r.Post("/notify", middleware.RequireAuthUser(h.sendReleaseNotification))
```

Comment:
In the end I used more action oriented route `/projects/{project_id}/releases/{release_id}/notify`

Reply:
After a quick search, couldn't find the official docs, but it seems like there is a strong consensus in the community to stick to nouns.
Would love to find the opposite evidence, though.

And while now it might seem too artificial to name it smth like `POST notification`, it actually might be a persistent resource later on (for example, we might decide to store these notification triggers for analytics purposes).

Reply:
Ok, I would choose `/slack-notification`

Reply:
Wow, I can edit other people's comments 😄 

I just wanted to add - let's be 100% RESTful and go with plural. 👌

Reply:
Ok, good point. Thanks!

If there is no other think you would change, please approve it :) thank you
I will update the route before merging.